src/BenchmarksClient/Startup.cs

```diff
@@ -96,8 +96,13 @@ public static int Main(string[] args)
                 var job = allJobs.FirstOrDefault();
                 if (job != null)
                 {
-                    var jobLogText = $"[ID:{job.Id} Connections:{job.Connections} Threads:{job.Threads} " +
-                        $"Duration:{job.Duration} Pipeline:{job.PipelineDepth} Method:{job.Method}";
+                    var jobLogText =
+                        $"[ID:{job.Id} Connections:{job.Connections} Threads:{job.Threads} Duration:{job.Duration} Method:{job.Method}";
+
+                    if (!string.IsNullOrEmpty(job.ScriptName) && job.PipelineDepth > 0)
```

Comment:
Allow scripts when the pipeline depth is `0`. At most, `Debug.Assert(!string.IsNullOrEmpty(job.ScriptName)` when `PipelineDepth > 0`.